.circleci/config.yml

```diff
@@ -1461,6 +1465,24 @@ jobs:
           working_directory: ./packages/contracts-bedrock
       - notify-failures-on-develop
 
+  publish-contract-artifacts:
```

Comment:
One thought is we should be fork friendly, ie not break in forks by:
- using top level variables that allows a small diff for forks
- putting things behind an `if repo name != ethereum-optimism/optimism: skip()`

Reply:
Good point - I can add that. Right now it'll show as "unauthorized" if a fork tries to run it.

Reply:
On second thought, I'll merge this now and add this in separately. I'm not sure there's a way to skip jobs if they need a context; I'll have to look into it.